packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -73,8 +78,12 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
            this.applySelectionZoom();
         }
         this.mouseIsDown = false;
-        if (!this.state.shiftKey) {
-            this.setState({ cursor: 'default'});
+        this.isSelecting = false;
+        if (!this.state.shiftKey && !this.state.ctrlKey) {
+            this.isPanning = false;
+            this.setState({ cursor: 'default' });
+        } else if (this.state.ctrlKey) {
```

Comment:
Cursor should go to grabbing if and only if ctrlKey is pressed.
Cursor should go to crosshair if and only if shiftKey is pressed.

Reply:
Fixed.